tests/integration-service/const.go

```diff
@@ -0,0 +1,43 @@
+package integration
+
+import (
+	"fmt"
+
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+)
+
+const (
+	componentRepoURL = "https://github.com/redhat-appstudio-qe/" + helloWorldComponentGitSourceRepoName
+	EnvNameForNBE    = "user-picked-environment"
+	revisionForNBE   = "main"
+	pathInRepoForNBE = "pipelines/integration_test_app.yaml"
+
+	BundleURL              = "quay.io/redhat-appstudio/example-tekton-bundle:integration-pipeline-pass"
+	BundleURLFail          = "quay.io/redhat-appstudio/example-tekton-bundle:integration-pipeline-fail"
+	InPipelineName         = "integration-pipeline-pass"
+	InPipelineNameFail     = "integration-pipeline-fail"
+	EnvironmentName        = "development"
+	gitURL                 = "https://github.com/redhat-appstudio/integration-examples.git"
+	revision               = "843f455fe87a6d7f68c238f95a8f3eb304e65ac5"
+	pathInRepo             = "pipelines/integration_resolver_pipeline_pass.yaml"
+	autoReleasePlan        = "auto-releaseplan"
+	targetReleaseNamespace = "default"
+
+	helloWorldComponentGitSourceRepoName = "hacbs-test-project"
```

Comment:
Please try to use a different repo (you may fork it with other name) for integration test, it might can cause conflict when running parallel with `rhtap-demo` test

Reply:
updated the test to now use a different (renamed) repo: `hacbs-test-project-integration` under the same org.